pkg/kubelet/cm/cpumanager/cpu_manager.go

```diff
@@ -129,19 +139,26 @@ func (s *sourcesReadyStub) AllReady() bool          { return true }
 func NewManager(cpuPolicyName string, reconcilePeriod time.Duration, machineInfo *cadvisorapi.MachineInfo, specificCPUs cpuset.CPUSet, nodeAllocatableReservation v1.ResourceList, stateFileDirectory string, affinity topologymanager.Store) (Manager, error) {
 	var topo *topology.CPUTopology
 	var policy Policy
+	var onlineCPUs cpuset.CPUSet
+	var err error
+
+	onlineCPUs, err = topology.OnlineCPUs()
```

Comment:
you can get cpus from machineInfo
also you don't take into account reservedCPU, as a result it's a capacity but not allocatable. But we need to know which one cpu kubelet will never use for cpu allocation.

Reply:
Does machineInfo take into account actually online CPUs?
good point about reserved cpus, will handle that

Reply:
Yes it does: https://github.com/google/cadvisor/blob/9b7851c9fb0efb02e330ec88a8b54569e7a87529/utils/sysfs/sysfs.go#L338

So we can get rid of this code, and I'll take into account the `reserved cpus` like @AlexeyPerevalov suggested/